validator_client/src/validator_store.rs

```diff
@@ -531,6 +533,52 @@ impl<T: SlotClock + 'static, E: EthSpec> ValidatorStore<T, E> {
         }
     }
 
+    pub async fn sign_blobs(
+        &self,
+        validator_pubkey: PublicKeyBytes,
+        blob_sidecars: BlobSidecarList<E>,
+        current_slot: Slot,
+    ) -> Result<SignedBlobSidecarList<E>, Error> {
+        let mut signed_blob_sidecars = Vec::new();
+
+        for blob_sidecar in blob_sidecars.into_iter() {
+            let slot = blob_sidecar.slot;
+
+            // Make sure the blob slot is not higher than the current slot to avoid potential attacks.
+            if slot > current_slot {
+                warn!(
+                    self.log,
+                    "Not signing blob with slot greater than current slot";
+                    "blob_slot" => slot.as_u64(),
+                    "current_slot" => current_slot.as_u64()
+                );
+                return Err(Error::GreaterThanCurrentSlot { slot, current_slot });
+            }
```

Comment:
I think we can remove this because we have the same check for the block and the block and blob slots should be equal or the proposal will be missed. I'm also not sure what the original intent of this check was.

Reply:
Removed, thanks!